lectures/inequality.md

```diff
@@ -362,53 +448,166 @@ ginis = []
+    caption: Gini coefficients of simulated data
     name: gini_simulated
   image:
     alt: gini_simulated
 ---
-plot_inequality_measures(σ_vals, 
-                         ginis, 
-                         'simulated', 
-                         '$\sigma$', 
-                         'gini coefficients')
+fix, ax = plot_inequality_measures(σ_vals, 
+                                  ginis, 
+                                  'simulated', 
+                                  '$\sigma$', 
+                                  'gini coefficients')
+plt.show()
 ```
 
 The plots show that inequality rises with $\sigma$, according to the Gini
 coefficient.
 
-+++
-
 ### Gini coefficient dynamics for US data
 
-Now let's look at Gini coefficients for US data derived from the SCF.
+Now let's look at Gini coefficients for US data. 
 
-The following code creates a list called ``Ginis``.
+In this section we will get pre-computed Gini coefficients from the World Bank using the [wbgapi](https://blogs.worldbank.org/opendata/introducing-wbgapi-new-python-package-accessing-world-bank-data).
 
- It stores data of Gini coefficients generated from the dataframe ``df_income_wealth`` and method [gini_coefficient](https://quanteconpy.readthedocs.io/en/latest/tools/inequality.html#quantecon.inequality.gini_coefficient), from [QuantEcon](https://quantecon.org/quantecon-py/) library.
+Let's search the world bank data for gini to find the Series ID.
 
 ```{code-cell} ipython3
-:tags: [hide-input]
+wb.search("gini")
+```
+
+We now know the series ID is `SI.POV.GINI`. 
+
+```{tip}
+Another, and often useful way to find series ID, is to use the [World Bank data portal](https://data.worldbank.org) and then use `wbgapi` to fetch the data.
+```
+
+Let us fetch the data for the USA.
+
+```{code-cell} ipython3
+data = wb.data.DataFrame("SI.POV.GINI", "USA")
+```
+
+```{code-cell} ipython3
+data
+```
+
+```{tip}
+This package often returns data with year information contained in the columns. This is not always convenient for simple plotting with pandas so it can be useful to transpose the results before plotting
+```
+
+```{code-cell} ipython3
+data = data.T
+data_usa = data['USA']
+```
+
+```{code-cell} ipython3
+fig, ax = plt.subplots()
+ax = data_usa.plot(ax=ax)
+ax.set_ylim(0,data_usa.max()+5)
+plt.show()
+```
+
+The gini coefficient is relatively slow moving and does not have significant variation in the full range from 0 to 100.
+
+Using `pandas` we can take a quick look across all countries and all years in the world bank dataset to understand how the Gini coefficient varies across countries and time.
+
+```{code-cell} ipython3
+gini_all = wb.data.DataFrame("SI.POV.GINI")
+```
+
+```{code-cell} ipython3
+# Create a long series with a multi-index of the data to get global min and max values
+gini_all = gini_all.unstack(level='economy').dropna()
+```
+
+```{code-cell} ipython3
+gini_all.plot(kind="hist", title="Gini coefficient");
+```
```

Comment:
The bar graph might look better with a few more bins.  And maybe outlines to show the bars?  I don't love the default bar chart in matplotlib...

Lowercase plot labels needed